### DISCUSSION TITLE
Configuration provider was specified but registry not enabled

### DISCUSSION THREAD

**Discussion by bighb69738** [edgexfoundry/community#208] [Q&A] [answered]
level=WARNING ts=2023-11-22T09:15:12Z app=device-template msg="Configuration provider was specified but registry not enabled"
level=INFO ts=2023-11-22T09:15:12Z app=device-template msg="iot_threadpool_alloc (threads: 8 max_jobs: 0 default_priority: -1 affinity: -1)"
level=INFO ts=2023-11-22T09:15:12Z app=device-template msg="iot_scheduler_alloc (priority: -1 affinity: -1)"
/device-example-c: Unrecognized option --confdir

How to fix the error

**Comment by cloudxxx8** [comment, marked as answer]
You can remove the --confdir from your start up command.
